CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# UNRELEASED
+
+- Dependency Updates:
```

Comment:
let's clarify this to mention these are dev dependencies. Also don't forget the Typescript version bump